src/mqt/predictor/ml/predictor.py

```
@@ -454,13 +797,14 @@ def _get_prepared_training_data(self) -> TrainingData:
 
 
 def predict_device_for_figure_of_merit(
-    qc: Path | QuantumCircuit, figure_of_merit: figure_of_merit = "expected_fidelity"
+    qc: Path | QuantumCircuit, figure_of_merit: figure_of_merit = "expected_fidelity", gnn: bool = False
```

Ruff (0.14.8)

Boolean-typed positional argument in function definition

(FBT001)